### ISSUE TITLE
Pylint doesn't know that hashlib's SHAKE hashers take a positional argument in their hexdigest

### ISSUE THREAD

**Issue by timmc-edx** [PyCQA/pylint#4039]
### Steps to reproduce
Run pylint on this file:

```python
import hashlib

hasher = hashlib.shake_128("test".encode())
output = hasher.hexdigest(16)
print(output)
```

### Current behavior

Emits linter error regarding hexdigest:
```
lint.py:4:9: E1121: Too many positional arguments for method call (too-many-function-args)
```

### Expected behavior

No linter error for that line. Unlike the other hash functions, SHAKE-128 (and SHAKE-256) have variable length output, so their hexdigest methods take a positional argument specifying the desired number of output bytes.

### pylint --version output
```
$ pylint --version
pylint 2.6.0
astroid 2.4.2
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0]
```

**Comment by hippo91** [comment]
@timmc-edx thanks for the report. I can reproduce it. Seems like the `astroid`'s brain for `hashlib` doesn't take into account 
SHAKE variable length digest.

**Comment by mbyrnepr2** [comment]
Can be closed when https://github.com/PyCQA/astroid/milestone/60 is available.

**Comment by mbyrnepr2** [comment]
> Can be closed when https://github.com/PyCQA/astroid/milestone/60 is available.
